### PR TITLE
Fix RUN_AFTER report function names

### PR DESCRIPTION
**Pull request by OHerasym**
Related:
        [APPLINK-16232](https://adc.luxoft.com/jira/browse/APPLINK-16232)

[old pull request](https://github.com/CustomSDL/sdl_atf/pull/91)

### CODE REVIEW

**Comment by OHerasym** [comment]
don't merge

**Comment by LuxoftAKutsan** [comment]
> don't merge
What do you mean?

File: modules/connecttest.lua [hunk 1]
```diff
@@ -188,8 +188,13 @@ end
 
 module.timers = { }
 
-function RUN_AFTER(func, timeout)
-  xmlReporter.AddMessage(debug.getinfo(1, "n").name, tostring(func), {["Timeout"] = tostring(timeout)})
+function RUN_AFTER(func, timeout, funcName)
+  func_name_str = "noname"
```
Comment (LuxoftAKutsan) [inline modules/connecttest.lua]: I con't like raw strings there.

File: modules/connecttest.lua [hunk 2]
```diff
@@ -252,8 +252,12 @@ function EXPECT_EVENT(event, name)
   return ret
 end
 
-function RAISE_EVENT(event, data)
-  xmlReporter.AddMessage(debug.getinfo(1, "n").name, tostring(event))
+function RAISE_EVENT(event, data, eventName)
+  event_str = "noname"
```
Comment (LuxoftAKutsan) [inline modules/connecttest.lua]: Same raw string. Constant would be useful

**Comment by LuxoftAKutsan** [comment]
Please describe problem in commit message.

File: modules/connecttest.lua (hunk 1 above)
Comment (OHerasym) [inline modules/connecttest.lua]: @LuxoftAKutsan , this strings is in different methods, in future default names can be different.

**Comment by OHerasym** [comment]
@LuxoftAKutsan , Done

**Comment by LuxoftAKutsan** [comment]
Reviewed. @AGaliuzov  please merge

**Pull request merged**